metsrw/premis_object.py

```diff
@@ -0,0 +1,146 @@
+import os
+
+from . import exceptions
+from . import utils
+
+
+class Object(object):
+    """
+    An object representing a PREMIS object element.
+
+    :param str filename: Filename.
+    :param object characteristics: :class:`ObjectCharacteristics` instance that is part of this PREMIS object.
+    :param str identifier_type: Identifier type.
+    :param str identifier_value: Identifier value.
+    :param str relationship_type: Relationship type.
+    :param str relationship_subtype: Relationship subtype.
+    """
+
+    def __init__(self, filename=None, characteristics=None, identifier_type=None, identifier_value=None, relationship_type=None, relationship_subtype=None):
```

Comment:
Is it a valid object if any or all of these attributes are None? What attributes are required in the PREMIS spec? Repeatable?